src/wallet/test/fuzz/coincontrol.cpp

```diff
@@ -43,40 +43,70 @@ FUZZ_TARGET(coincontrol, .init = initialize_coincontrol)
-                (void)coin_control.HasSelected();
-            },
-            [&] {
-                (void)coin_control.IsSelected(out_point);
-            },
-            [&] {
-                (void)coin_control.IsExternalSelected(out_point);
-            },
             [&] {
                 (void)coin_control.GetExternalOutput(out_point);
             },
             [&] {
                 (void)coin_control.Select(out_point);
+                assert(coin_control.IsSelected(out_point));
             },
             [&] {
                 const CTxOut tx_out{ConsumeMoney(fuzzed_data_provider), ConsumeScript(fuzzed_data_provider)};
-                (void)coin_control.Select(out_point).SetTxOut(tx_out);
+                auto& input = coin_control.Select(out_point);
+                input.SetTxOut(tx_out);
+                assert(input.HasTxOut());
+                assert(input.GetTxOut() == tx_out);
+                assert(coin_control.IsExternalSelected(out_point));
             },
             [&] {
                 (void)coin_control.UnSelect(out_point);
+                assert(!coin_control.IsSelected(out_point));
             },
             [&] {
                 (void)coin_control.UnSelectAll();
+                assert(!coin_control.HasSelected());
             },
             [&] {
-                (void)coin_control.ListSelected();
+                const std::vector<COutPoint> selected = coin_control.ListSelected();
+                assert(selected.empty() == !coin_control.HasSelected());
+                for (const auto& out : selected)
+                    assert(coin_control.IsSelected(out));
             },
             [&] {
                 int64_t weight{fuzzed_data_provider.ConsumeIntegral<int64_t>()};
                 (void)coin_control.SetInputWeight(out_point, weight);
+                assert(coin_control.GetInputWeight(out_point) == weight);
+            },
+            [&] {
+                (void)coin_control.GetSequence(out_point);
+            },
+            [&] {
+                (void)coin_control.GetScripts(out_point);
+            },
+            [&] {
+                uint32_t sequence{fuzzed_data_provider.ConsumeIntegral<uint32_t>()};
+                (void)coin_control.Select(out_point).SetSequence(sequence);
+                assert(coin_control.Select(out_point).GetSequence() == sequence);
+            },
+            [&] {
+                const CScript script{ConsumeScript(fuzzed_data_provider)};
+                (void)coin_control.Select(out_point).SetScriptSig(script);
+                assert(coin_control.Select(out_point).HasScripts());
+                const auto& scripts = coin_control.Select(out_point).GetScripts();
+                assert(scripts.first && *scripts.first == script);
+            },
+            [&] {
+                const CScriptWitness script_wit{ConsumeScriptWitness(fuzzed_data_provider)};
+                (void)coin_control.Select(out_point).SetScriptWitness(script_wit);
```

Comment:
`SetScriptWitness` is already void:

```suggestion
                coin_control.Select(out_point).SetScriptWitness(script_wit);
```